test/main.ml

```diff
@@ -276,6 +277,42 @@ module Tests (Backend : Guard.Persistence_s) = struct
           (Ok true)
   ;;
 
+  let operator_works ?ctx _ () =
+    let%lwt actual =
+      let open Lwt_result.Syntax in
+      (* Note: a user can be an actor or a target, so 'to_authorizable' has to
+         be called for both roles *)
+      let* _ = User.to_authorizable ?ctx hugo in
+      let* target' = UserTarget.to_authorizable ?ctx hugo in
+      let* actor = User.to_authorizable ?ctx thomas in
+      let* () =
+        Backend.Actor.grant_roles
+          ?ctx
+          actor.Guard.Authorizable.uuid
+          (Guard.ActorRoleSet.singleton
+             (`Editor target'.Guard.AuthorizableTarget.uuid))
+      in
+      let* actor = User.to_authorizable ?ctx thomas in
```

Comment:
I also don't quite understand why a second call of this is necessary (Timo and I had this question before once)

Reply:
Creating a new issue => Resolving